src/content/apis/sports/api-nba.md

```diff
@@ -0,0 +1,10 @@
+---
+name: API NBA
+description: All NBA Stats DATA, Games, Livescore, Standings, Players, Statistics…
+categories:
+  - sports
+features:
+  https: true
+url: https://api-sports.io/documentation/nba/v2
+logo: https://www.api-basketball.com/public/img/home1/hero-banner1.png

```

Comment:
The logo has to be downloaded and place inside the repository itself.
Please check the contributing guides for more details.